src/components/pages/UpdateFarmer/data.js

```diff
@@ -0,0 +1,58 @@
+export default {
```

Comment:
Do we still need this file? correct me if am wrong. I guess the data is now coming from the farmer's table

Reply:
Nope, will delete. 👍